Modules/Plugins/Armory.lua

```diff
@@ -502,22 +508,25 @@ function A:UpdatePageStrings(_, slotId, _, slotItem, slotInfo, which)
         local text = slotInfo.enchantTextShort
         if self.db.pageInfo.abbreviateEnchantText then text = self:EnchantAbbreviate(slotInfo.enchantText) end
         if slotOptions.direction == self.enumDirection.LEFT then
-          slotItem.enchantText:SetText(F.String.FastGradient(text, 0, 0.69, 0.11, 0.07, 0.90, 0.15))
+          slotItem.enchantText:SetText(F.String.FastGradient(text, A.darkGreen.r, A.darkGreen.g, A.darkGreen.b, A.lightGreen.r, A.lightGreen.g, A.lightGreen.b))
```

Comment:
Not a fan of such a long argument list. Can we somehow do `F.String.FastGradient(text, var1, var2)` ?

Reply:
I can try combining this together with the earlier comment about using `HexToRGB` with the function: `FastGradientHex()` that already exists in `F.String` that would then shorten the argument list.
I personally am of the opinion that descriptive arguments are better than magic numbers, as it will self document that way and be easier to understand for people.